### PR TITLE
[DOCFIX] Improve Contributor Tools documentation

### PR DESCRIPTION
**Pull request by dbw9580**
Improve documentation on contributor tools.

### CODE REVIEW

File: docs/en/contributor/Contributor-Tools.md [hunk 1]
```diff
@@ -275,7 +316,7 @@ RPCs on clients and implementing the RPCs on servers. To regenerate Java code af
 a gRPC definition, you must rebuild `alluxio-core-transport` module with `'generate'` maven profile.
 
 ```console
-$ mvn clean install -Pgenerate
+$ mvn clean install -Pgenerate -pl "org.alluxio:alluxio-core-transport"
```
Comment (dbw9580) [inline docs/en/contributor/Contributor-Tools.md:319]: Building only the transport module saves a lot of time.

**Review by dbw9580 — COMMENTED** [review]
Added some comments for reviewers.

File: docs/en/contributor/Contributor-Tools.md [hunk 2]
```diff
@@ -39,39 +39,62 @@ To use IntelliJ IDEA to contribute to Alluxio, simply open IntelliJ and select "
...
-3. Edit `conf/alluxio-site.properties`
-```console
-  alluxio.home={alluxio.home}
-  alluxio.master.hostname=localhost
-  alluxio.job.master.hostname=localhost
-  alluxio.master.journal.type=UFS
-```
+3. Edit `conf/alluxio-site.properties` to contain these configurations
+   ```console
+   alluxio.home=/path/to/alluxio
```
[the remark this answers is not in the document]
Comment (Xenorith) [reply docs/en/contributor/Contributor-Tools.md:80]: sounds good. this is what i see more commonly as well

File: docs/en/contributor/Contributor-Tools.md [hunk 3]
```diff
@@ -80,58 +103,75 @@ You will need to mark the directory as "Generated Sources Root" for IntelliJ to
...
+    ```console
+    alluxio.home=/path/to/alluxio
+    alluxio.master.hostname=localhost
+    alluxio.job.master.hostname=localhost
+    alluxio.master.embedded.journal.addresses=localhost:19200,localhost:19201,localhost:19202
+    alluxio.master.rpc.addresses=localhost:19998,localhost:19988,localhost:19978
+    ```
+   The ports are defined in the run configurations.
+5. In Intellij, start the Alluxio master processes by selecting `Run > Run > 
+   AlluxioMaster-0`, `AlluxioMaster-1` and `AlluxioMaster-2`
```
Comment (Xenorith) [inline docs/en/contributor/Contributor-Tools.md:128]:
```suggestion
   AlluxioMaster-0`, `AlluxioMaster-1`, and `AlluxioMaster-2`
```

File: docs/en/contributor/Contributor-Tools.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/contributor/Contributor-Tools.md:319]: 👍

**Review by Xenorith — APPROVED** [review]
looks good overall, also thanks for all the formatting changes

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**